canvas_manage_course/urls.py

```diff
@@ -15,3 +16,9 @@
     url(r'^not_authorized$', icommons_ui_views.not_authorized, name='not_authorized'),
     url(r'^tool_config$', views.tool_config, name='tool_config'),
 ]
+
+if settings.DEBUG:
+    import debug_toolbar
```

Comment:
Can we add put this in a try/except block?    We normally don't require the toolbar in our aws environments, although we might want to run in `DEBUG` mode on a non-prod deployed instance.